app/src/main/java/org/fossasia/openevent/general/favorite/FavoriteFragment.kt

```diff
@@ -70,6 +83,20 @@ class FavoriteFragment : Fragment() {
                 showEmptyMessage(favoriteEventsRecyclerAdapter.itemCount)
             }
         }
+        val calendar = Calendar.getInstance()
```

Comment:
No need to create calendar here and then pass to viewmodel. Its counterintuitive.